From 0be67ded0a46901554df52c6dff6a242f890ff0c Mon Sep 17 00:00:00 2001
From: TANIGUCHI Takaki <takaki@debian.org>
Date: Thu, 8 Oct 2015 11:16:20 -0700
Subject: <short summary of the patch>

 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 python-botocore (0.81.0-1) UNRELEASED; urgency=medium
 .
   * New upstream release

Patch-Name: remove-duplicated-code
---
 botocore/awsrequest.py   | 12 ++++++------
 botocore/compat.py       |  2 +-
 botocore/endpoint.py     |  4 ++--
 botocore/retryhandler.py |  4 ++--
 botocore/utils.py        |  2 +-
 5 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/botocore/awsrequest.py b/botocore/awsrequest.py
index bf4a4da..ccc01a6 100644
--- a/botocore/awsrequest.py
+++ b/botocore/awsrequest.py
@@ -19,14 +19,14 @@ import socket
 import inspect
 
 from botocore.compat import six
-from botocore.vendored.requests import models
-from botocore.vendored.requests.sessions import REDIRECT_STATI
+from requests import models
+from requests.sessions import REDIRECT_STATI
 from botocore.compat import HTTPHeaders, HTTPResponse
 from botocore.exceptions import UnseekableStreamError
-from botocore.vendored.requests.packages.urllib3.connection import VerifiedHTTPSConnection
-from botocore.vendored.requests.packages.urllib3.connection import HTTPConnection
-from botocore.vendored.requests.packages.urllib3.connectionpool import HTTPConnectionPool
-from botocore.vendored.requests.packages.urllib3.connectionpool import HTTPSConnectionPool
+from urllib3.connection import VerifiedHTTPSConnection
+from urllib3.connection import HTTPConnection
+from urllib3.connectionpool import HTTPConnectionPool
+from urllib3.connectionpool import HTTPSConnectionPool
 
 
 logger = logging.getLogger(__name__)
diff --git a/botocore/compat.py b/botocore/compat.py
index b424b91..e8a1323 100644
--- a/botocore/compat.py
+++ b/botocore/compat.py
@@ -16,7 +16,7 @@ import datetime
 import sys
 import inspect
 
-from botocore.vendored import six
+import six
 
 
 if six.PY3:
diff --git a/botocore/endpoint.py b/botocore/endpoint.py
index 963068d..9409499 100644
--- a/botocore/endpoint.py
+++ b/botocore/endpoint.py
@@ -17,8 +17,8 @@ import logging
 import time
 import threading
 
-from botocore.vendored.requests.sessions import Session
-from botocore.vendored.requests.utils import get_environ_proxies
+from requests.sessions import Session
+from requests.utils import get_environ_proxies
 
 import botocore.response
 import botocore.exceptions
diff --git a/botocore/retryhandler.py b/botocore/retryhandler.py
index 2505db2..c12d2e0 100644
--- a/botocore/retryhandler.py
+++ b/botocore/retryhandler.py
@@ -17,8 +17,8 @@ import functools
 import logging
 from binascii import crc32
 
-from botocore.vendored.requests import ConnectionError, Timeout
-from botocore.vendored.requests.packages.urllib3.exceptions import ClosedPoolError
+from requests import ConnectionError, Timeout
+from urllib3.exceptions import ClosedPoolError
 
 from botocore.exceptions import ChecksumError
 
diff --git a/botocore/utils.py b/botocore/utils.py
index 7ae0ae3..c36be74 100644
--- a/botocore/utils.py
+++ b/botocore/utils.py
@@ -22,7 +22,7 @@ from dateutil.tz import tzlocal, tzutc
 
 from botocore.exceptions import InvalidExpressionError, ConfigNotFound
 from botocore.compat import json, quote, zip_longest
-from botocore.vendored import requests
+import requests
 from botocore.compat import OrderedDict
 
 
